.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -32,19 +32,30 @@ body:
       label: Backtrace
       description: If your bug produces a backtrace, please include it here.
       render: shell
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant Logs
+      description: If possible, reproduce the bug with the envrionment variable RUST_LOG set to 'trace', and attach the printed logs
+      render: shell
   - type: input
     id: os_version
     attributes:
       label: Your operating system and version
     validations:
       required: true
   - type: input
-    id: node_version
+    id: binary_type
     attributes:
-      label: Your Node version (`node --version`)
-      placeholder: ex. v16.13.1
+      label: Local process
+      description: What process are you trying to run with mirrord?
```

Comment:
Add "please run `file $(which PROCESSNAME)`" as it's very important for debugging